aten/src/ATen/native/TensorShape.cpp

```diff
@@ -3169,6 +3186,34 @@ Tensor squeeze_quantized(const Tensor& self, int64_t dim) {
   return squeeze_qtensor(self, dim);
 }
 
+Tensor squeeze(const Tensor& self, IntArrayRef dims) {
+  auto mask = dim_list_to_bitset(dims, self.dim());
+  auto g = inferSqueezeGeometry(self, mask);
+  at::Tensor result = self.as_strided_symint(std::get<0>(g), std::get<1>(g));
+  auto maybe_outnames = namedinference::compute_squeeze_outnames(self, mask);
+  namedinference::propagate_names_if_nonempty(result, maybe_outnames);
+  return result;
+}
+
+Tensor squeeze_multiple(const Tensor& self, IntArrayRef dims) {
```

Comment:
Why is this not the `squeeze()` function above?

Reply:
I've updated the `NestedTensor` and `Quantized` paths to support the multiple dims, so this is no longer necessary.